server/src/controllers/profile.controllers/changeEmail.ts

```diff
@@ -0,0 +1,21 @@
+import {NextFunction, Request, Response} from 'express';
+import validateInput, {IValidationError} from '../../helpers/validation';
+import {sendChangingEmail} from '../../helpers/send.email';
+import {changeEmailMessage} from '../../helpers/send.email.change.email';
+import Validator from 'validator';
+
+export const changeEmail = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+    try {
+        const { newEmail, profileUser: {email} }: {newEmail: string, profileUser: {email: string}} = req.body;
```

Comment:
pls create interface for {newEmail: string, profileUser: {email: string}}
it is hard to read

Reply:
Fixed